package.json

```diff
@@ -18,7 +18,8 @@
   },
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "dependencies": {
-    "classnames": "^2.2.5"
+    "classnames": "^2.2.5",
+    "react-bootstrap": "^0.30.8"
   },
```

Comment:
@jtomasek i guess my thinking there is it would be up to the consumer to ensure that the optional dependency is installed (no need to add checks in the code). I don't think we would code for the case that a dependency is missing or not available. It's more for the build size I mentioned in the case you are not using React-Bootstrap components.

Again, no issue here though w/ @jtomasek's approach of making this a hard dependency. I just wanted to clarify our stance on npm dependencies early on. We will run into the same thing w/ other third party libs.